server/sonar-server/src/main/java/org/sonar/server/issue/filter/IssueFilterParameters.java

```diff
@@ -70,6 +70,8 @@
   public static final String ASC = "asc";
   public static final String IGNORE_PAGING = "ignorePaging";
 
+  public static final String FACET_ASSIGNED_TO_ME = "assigned_to_me";
```

Comment:
I'm not suer it's the best place to put this constant... Where are other constants on facets defined ?